Makefile

```diff
@@ -1,2 +1,56 @@
+all: whatdoesitdo tests authors
+
 whatdoesitdo:
+	@# generate the README "What does it do?" section
 	@./gen-readme.sh
+
+# To decrease tests verbosity, comment out unneeded targets
+tests: downloadffprefs checknotcovered checkdeprecated stats cleanup
```

Comment:
> Is it so, that everything below that will not run as parallel or what?

This is what it looks like, but I have no experience with parallel usage of Make - very interesting, I really need to read the [full Make manual](https://www.gnu.org/software/make/manual/make.html).

Unfortunately no time to work on it at the moment, would you be ok to apply the global `MAKEFLAGS := --jobs=1` var, and open an issue (`Makefile: parallelize tests`) ? We can improve this later.